From 6c8e78fb66d35250eb7c69fee041424a16721f7c Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Tue, 21 Jan 2025 19:53:41 +0100
Subject: [PATCH] Add missing include for uint64_t

With gcc 15 it is necessary to explicitly include <cstdint> for
uint64_t.
---
 src/global/utils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/global/utils.h b/src/global/utils.h
index 74c16235..a52f932a 100644
--- a/src/global/utils.h
+++ b/src/global/utils.h
@@ -4,6 +4,7 @@
 // Author: Nils Schimmelmann <nschimme@gmail.com> (Jahara)
 
 #include <algorithm>
+#include <cstdint>
 #include <cstring>
 #include <memory>
 #include <optional>
-- 
2.48.1

